tests/Aspire.Playground.Tests/ProjectSpecificTests.cs

```diff
@@ -20,10 +21,10 @@ public async Task WithDockerfileTest()
         await using var app = await appHost.BuildAsync();
 
         await app.StartAsync();
-        await app.WaitForResources().WaitAsync(TimeSpan.FromMinutes(2));
+        await app.WaitForResources().WaitAsync(TestConstants.ExtraLongTimeoutTimeSpan);
 
         await app.WaitForTextAsync($"I'm Batman. - Batman")
-                .WaitAsync(TimeSpan.FromMinutes(3));
+                .WaitAsync(TestConstants.ExtraLongTimeoutTimeSpan);
```

Comment:
this ought to be just LongTimeoutTimeSpan as it's not pulling resources, but that would be less than 3 so..